pkg/providers/pricing/skus_missing_price.go

```diff
@@ -0,0 +1,100 @@
+// Portions Copyright (c) Microsoft Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pricing
+
+// setting to a high value means it won't be chosen automatically,
+// but will still be available if chosen explicitly
+const missingPrice float64 = 999.0
+
+// skusWithMissingPrice is a map of SKUs that are known to have missing prices in the Azure pricing API.
+var skusWithMissingPrice = map[string]float64{
```

Comment:
This still doesn't necessarily catch all the case, since some skus might only be supported in one region, for instance?

I don't really see a harm in adding this in though, since with such high pricing it should only select these if the selectors are only for skus with missing prices.

Does feel a bit like a more complete assessment of skus might be worth it though.

Reply:
SKUs that are not supported in a particular region should already be filtered out on the skewer / resource API side, andn won't have available offerings, so should not be a problem.

Reply:
Ah, make sense. Good point.